apis/nucleus/src/index.js

```diff
@@ -426,6 +436,9 @@ function nuked(configuration = {}) {
            * @param {boolean=} [options.dense=false] Reduces padding and text size (not applicable for existing objects)
            * @param {string=} [options.stateName="$"] Sets the state to make selections in (not applicable for existing objects)
            * @param {object=} [options.properties={}] Properties object to extend default properties with
+           * @param {OnSelectionConfirm=} [options.onSelectionConfirm=()=>{}] Callback function when confirm happens
+           * @param {OnSelectionCancel=} [options.OnSelectionCancel=()=>{}] Callback function when cancel happens
```

Comment:
Should be able to spec these as functions directly without adding a new type.

Reply:
Also, might be better to have this as an event API instead. So you would go:
```
FieldInstance.on("selectionsCancel", ()=> { ... })
```
instead, giving us a bit more flexibility instead of just adding more and more callbacks.

Reply:
> Should be able to spec these as functions directly without adding a new type.

You mean by using ignore in their types like this example?: https://github.com/qlik-oss/nebula.js/blob/6695ee87314e095a040da7132f528f01a91d5e28/apis/nucleus/src/index.js#L115-L117

Reply:
> Should be able to spec these as functions directly without adding a new type.

I removed them from types because we are no longer passing it manually, but do we still ned them?